packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -573,6 +666,10 @@ export class RunningSummarizer implements IDisposable {
 				this.beforeSummaryAction();
 			},
 			async () => {
+				if (this.mc.config.getBoolean("Fluid.Summarizer.TryDynamicRetries")) {
```

Comment:
After you merge your other PR to main and here, I think it would be nice to pull the existing logic into a function as well, so this code is symmetrical - calls one or the other helper, then wraps up (might as well move `this.stopSummarizerCallback` out too, and anything else common).